TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -905,6 +909,147 @@ private void WritePeptideResults()
+                var culture = CultureInfo.CurrentCulture;
+                foreach (var variant in variants)
+                {
+                    bool intersects = PsmTsvWriter.IntersectsWithVariation(variantPWSM.Peptide, variant, true);
+                    if(intersects == true)
+                    {
+                        if (culture.CompareInfo.IndexOf(variant.Description.Description, "missense_variant", CompareOptions.IgnoreCase)>=0)
+                        {
+                            savCount++;
+                        }
+                        if (culture.CompareInfo.IndexOf(variant.Description.Description, "frameshift_variant", CompareOptions.IgnoreCase) >= 0) 
+                        {
+                            frameshiftCount++;
+                        }
+                        if (culture.CompareInfo.IndexOf(variant.Description.Description, "stop_gained", CompareOptions.IgnoreCase) >= 0)
+                        {
+                            stopGainCount++;
+                        }
+                        if ((culture.CompareInfo.IndexOf(variant.Description.Description, "conservative_inframe_insertion", CompareOptions.IgnoreCase) >= 0) || (culture.CompareInfo.IndexOf(variant.Description.Description, "disruptive_inframe_insertion", CompareOptions.IgnoreCase) >=0))
+                        {
+                            insertionCount++;                            
+                        }
+                        if ((culture.CompareInfo.IndexOf(variant.Description.Description, "conservative_inframe_deletion", CompareOptions.IgnoreCase) >= 0) || (culture.CompareInfo.IndexOf(variant.Description.Description, "disruptive_inframe_deletion", CompareOptions.IgnoreCase) >= 0))
+                        {
+                            deletionCount++;
+                        }
+                        if (culture.CompareInfo.IndexOf(variant.Description.Description, "exon_loss_variant", CompareOptions.IgnoreCase) >= 0)
+                        {
+                            exonLossCount++;
+                        }
+                        if (culture.CompareInfo.IndexOf(variant.Description.Description, "stop_lost", CompareOptions.IgnoreCase) >= 0)
+                        {
+                            stopLossCount++;
+                        }
+
+                    }
+                }
+            }
+
+            string[] variantResults = new string[15];
+            variantResults[0] = "Variant Result Summary";
+            variantResults[2] = "--------------------------------------------------";
+            variantResults[4] = "Number of potentially variant containing peptides identified at 1% FDR: " + variantPeptides.Count();
+            variantResults[5] = "Number of unqiuely identified variant peptides at 1% FDR: " + confidentVariantPeps.Count();
+            variantResults[6] = "Number of SAV variant peptides at 1% FDR: " + savCount;
```

Comment:
does each psm get labeled as to why kind of variant it is? in one of the columns in the tsv?

Reply:
No it does not at this time